### PR TITLE
fix(bundle): fix CRD conversion with minimum validation field

### PR DESCRIPTION
**Pull request by njhale**
- Marshals to JSON then Unmarshals to CustomResourceDefinition when
updating bundle cache to avoid a FromUnstructured call.
- Runs `make vendor`.

### CODE REVIEW

File: pkg/registry/bundle.go [hunk 1]
```diff
@@ -205,7 +206,13 @@ func (b *Bundle) cache() error {
 	for _, o := range b.Objects {
 		if o.GetObjectKind().GroupVersionKind().Kind == "CustomResourceDefinition" {
 			crd := &apiextensions.CustomResourceDefinition{}
-			if err := runtime.DefaultUnstructuredConverter.FromUnstructured(o.UnstructuredContent(), crd); err != nil {
+			// Marshal Unstructured and Unmarshal as CustomResourceDefinition. FromUnstructured has issues
```
Comment (alecmerdler) [inline pkg/registry/bundle.go]: Can we file a bug upstream to document this for other devs?

Comment (njhale) [reply pkg/registry/bundle.go]: @alecmerdler yeah, this fix works for now but I need to nail down the exact wording. Maybe there's something additional we're doing wrong that's causing `FromUnstructured` to not work.

**Pull request merged**